sdks/android/machine.config

```diff
@@ -11,7 +11,7 @@
 		<section name="assemblyBinding"  type="System.Configuration.IgnoreSection, System.Configuration, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" allowLocation="false" />
 		<section name="satelliteassemblies" type="System.Configuration.IgnoreSection, System.Configuration, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" allowLocation="false" />
 		<section name="startup" type="System.Configuration.IgnoreSection, System.Configuration, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" allowLocation="false"/>
-		<section name="system.codedom" type="System.CodeDom.Compiler.CodeDomConfigurationHandler, System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+		<section name="system.codedom" type="System.CodeDom.Compiler.CodeDomConfigurationHandler, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
```

Comment:
this is wrong as System.dll isn't 4.0 on android (not sure how much it matters for this).

Reply:
mobile profiles don't have System.Configuration at all

Reply:
yeah which brings the point of why we have this file at all on mobile 😄

Reply:
so I guess we can safely remove it from android, right?

Reply:
It can certainly be simplified but I recall stuff like pubTokenMapping be used by runtime directly or something like that